spec/features/instructeurs/instruction_spec.rb

```diff
@@ -206,7 +208,7 @@ def log_out
   end
 
   def ask_confidential_avis(to, introduction)
-    fill_in 'avis_emails', with: to
+    page.execute_script("document.querySelector('#avis_emails').value = '#{to}'")
```

Comment:
@krichtof tu avais eu besoin de faire ça, toi, pour que les tests utilisent le `ComboMultipleDropdownList` ?

Reply:
J'ai même fait pire, pour arriver à faire passer un test : 

```
    # hack because for unknown reason, the click on input doesn't show combobox-popover with selenium driver
    script = "document.evaluate(\"//input[@aria-label='#{champ}']//ancestor::div[@data-reach-combobox]/div[@data-reach-combobox-popover]\", document, null, XPathResult.UNORDERED_NODE_ITERATOR_TYPE, null).iterateNext().removeAttribute(\"hidden\")"
    execute_script(script)
```

C'est par ici : https://github.com/betagouv/demarches-simplifiees.fr/blob/6845aa69f16ebcc2212b32631183de7665813a5d/spec/support/feature_helpers.rb#L111

Reply:
Hooo, je vois :)

Ok, donc ici on peut utiliser ton helper `select_multi`, si je comprends bien.

Reply:
Tout à fait !